Support aggregated issues

rimrul opened the pull request:
Our component monitoring can aggregate multiple issues into one if it would open multiple issues for the same component at the same time.  We need to treat these issues different from other issues, because their title doesn't contain version information.

This fixes #11

dscho commented:
Nice!

dscho commented:
@rimrul we do not even have to rename the title of https://github.com/git-for-windows/MSYS2-packages/pull/67 to avoid running into https://github.com/git-for-windows/gfw-helper-github-app/commit/b20304f0e2ed7f6c67e9784193985b9efcdf5f89 because the title does not match the expected format...

dscho commented:
> we do not even have to rename the title of [git-for-windows/MSYS2-packages#67](https://github.com/git-for-windows/MSYS2-packages/pull/67) to avoid running into [b20304f](https://github.com/git-for-windows/gfw-helper-github-app/commit/b20304f0e2ed7f6c67e9784193985b9efcdf5f89) because the title does not match the expected format...

Oh, I missed that [you renamed it](https://github.com/git-for-windows/MSYS2-packages/pull/67#event-8035988722)...

BTW unless I am mistaken, the current code would say `bash: update to 5.2.15` (not `.015`, as the currently-used version format would suggest). Do we want to address that, still, or is it okay to leave that as-is, or do we want to say `bash: update to 5.2 patchlevel 15`, what do you think?

rimrul commented:
> > we do not even have to rename the title of [git-for-windows/MSYS2-packages#67](https://github.com/git-for-windows/MSYS2-packages/pull/67) to avoid running into [b20304f](https://github.com/git-for-windows/gfw-helper-github-app/commit/b20304f0e2ed7f6c67e9784193985b9efcdf5f89) because the title does not match the expected format...
> 
> Oh, I missed that [you renamed it](https://github.com/git-for-windows/MSYS2-packages/pull/67#event-8035988722)...

I expected to fix up that PR by hand, when I changed it.

> BTW unless I am mistaken, the current code would say `bash: update to 5.2.15` (not `.015`, as the currently-used version format would suggest). Do we want to address that, still, or is it okay to leave that as-is, or do we want to say `bash: update to 5.2 patchlevel 15`, what do you think?

I don't quite know why I wrote `015` when I renamed it, maybe it was inspired by old build agent commits like 
https://github.com/git-for-windows/MSYS2-packages/commit/eb5103780fdc3d63678ee30ab18c4b4199ad8def, but honestly either of the tree is fine by me. All three of them convey the same information to a human reader and all three of them seem more or less equally obvious.

dscho commented:
I guess I have a slight preference for the one that mentions "patchlevel" explicitly.
